### PR TITLE
Configure lambdatest run arguments

### PR DESCRIPTION
**Pull request by rgagarinr**
## Summary:
Issue: FEI-4628

## Test plan:
Run job with Replay

### CODE REVIEW

File: jobs/e2e-test-cypress.groovy [hunk 1]
```diff
@@ -99,16 +99,14 @@ def runLamdaTest() {
    // TODO(ruslan): Implement TEST_TYPE param to use decorator or flag 
    // in cypress script files. 
    // TODO(ruslan): Use build tags --bt with prod/znd states.
+   // TODO(ruslan): Add retries flag.
```
Comment (jandrade) [inline jobs/e2e-test-cypress.groovy:102]: Oh so that breaks when using single quotes?

**Review by jandrade — APPROVED** [review]
Looks great!

File: jobs/e2e-test-cypress.groovy (hunk 1 above)
Comment (rgagarinr) [reply jobs/e2e-test-cypress.groovy:102]: yeah, I'm not sure why it's failing when [separating](https://docs.cypress.io/guides/guides/command-line#cypress-run-config-lt-config-gt) multiple values with commas. I'll check with the lambda team.

```
Cypress encountered an error while parsing the argument: --config

You passed: baseUrl="https://prod-xxyyy.khanacademy.org",

The error was: Cannot read properties of undefined (reading 'split')
```

**Comment by rgagarinr** [comment]
Thanks for helping @jandrade

**Pull request merged**